frontend/src/common/ConfirmationDialogue.tsx

```diff
@@ -15,6 +18,15 @@ type ConfirmationDialogProps = {
   errorMessage?: string
 }
 
+const Transition = forwardRef(function Transition(
```

Comment:
Consider putting the `transition` in a single place and importing it, as this code is repeated verbatim in half a dozen places.

Reply:
Thrown into utils folder